source/index.rst

```diff
@@ -6,10 +6,13 @@
 Mixxx |version| User Manual
 ***************************
 
-Mixxx is Free [#f1]_ DJ software for Windows, Mac OS X, and Linux. **Mixxx gives
-you everything you need to become a professional DJ** (except talent and hard
-work). Using Mixxx, you can rock the party with your vinyl turntables, your
-:term:`MIDI`/:term:`HID` controllers, or even just your keyboard.
+Mixxx is free [#f1]_ DJ software for Windows, Mac OS X and Linux. It's
+easy to use, so you will start mixing your favorite songs in no time,
+even if you've never DJed before. Mixxx is also
+powerful, so experienced DJs will find all the advanced features they
+would expect. You can perform with Mixxx using :term:`MIDI` and :term:`HID`
+controllers, CD players, vinyl turntables or even just your
+computer's keyboard and mouse.
```

Comment:
Your version is already better. I would change the second and third sentence tough.

>
It's easy to use so you can start mixing your favorite songs in no time. Mixxx offers a variety of features like effects , master-sync, vinyl-scratching and many more for experienced DJs and beginners alike.


If you also link to the features that would be nice. It's more personal and says that all features can be used independent if you are a pro or not.

 I would also scratch the `computer's` in the last sentence. It is not needed to understand what you mean.